Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/alpine:3.3
+
+ENTRYPOINT ["/bin/fluentd-kinesis-forwarder-monitor"]
+
+RUN apk-install ca-certificates
+
+COPY fluentd-kinesis-forwarder-monitor /bin/fluentd-kinesis-forwarder-monitor
```

Comment:
That's generally true in order to save time. However a entry point or cmd directive without an updated file does not make much sense. In other words it does not take time to set entrypoint workdir cmd etc so I prefer to position them based on readability